README.md

```diff
@@ -80,22 +94,30 @@ npm install
 npm run testrpc
 ```
 
-#### Terminal 3: Start the testing Python stub
+#### Terminal 3: Python tools
 
 ```bash
 # Configure your environment
 . ./setup_env.sh
 
 cd pyClient
-# Follow the few steps described in the README of the python stub
 ```
 
-### Use the pyClient
+Follow the steps described in the [pyClient README](pyClient/README.md) to run
+tests or invoke the zeth tools.
+
+### pyClient and jsClient
```

Comment:
This seems to be a old version of the README, the `jsClient` has been removed and all mentions of it removed from the README (see: https://github.com/clearmatics/zeth/#use-the-pyclient)

Reply:
Must have been a rebasing issue.  I've removed the section.